.clj-kondo/config.edn

```diff
@@ -194,7 +194,6 @@
     metabase.db.jdbc-protocols                                      mdb.jdbc-protocols
     metabase.db.metadata-queries                                    metadata-queries
     metabase.db.spec                                                mdb.spec
-    metabase.db.query                                               mdb.query
```

Comment:
Just removed this as I noticed the namespace doesn't exist anymore

Reply:
It does exist, it's right here https://github.com/metabase/metabase/blob/master/src/metabase/db/query.clj